scripts/felicitasSpecials.coffee

```diff
@@ -3,25 +3,35 @@
 #
 # Commands:
 #   woz felicitas specials
+#   woz felicitas specials (day of the week)
 
 Cheerio = require("cheerio")
 Moment = require("moment")
 
-url_prefix = "http://www.felicitas.ca/menu-item/"
-
-today = Moment().format("dddd").toLowerCase()
+urlPrefix = "http://www.felicitas.ca/menu-item/"
+daysOfTheWeek = ["sunday","monday", "tuesday",
+            "wednesday", "thursday", "friday", "saturday"]
 
 module.exports = (robot) ->
-  robot.respond /felicitas\s+-?specials/i, (msg) ->
+  robot.respond /felicitas\s+-?specials\s*(\w+)?/i, (msg) ->
+    response = ''
```

Comment:
Does this need to be instantiated here any more? 

I don't think it is being used, given that 
```coffeescipt
if ''
```
returns false